src/main/java/keycontacts/logic/parser/KeyContactsParser.java

```diff
@@ -67,6 +68,7 @@ public Command parseCommand(String userInput) throws ParseException {
         case MakeupLessonCommand.COMMAND_WORD -> new MakeupCommandParser().parse(arguments);
         case ScheduleCommand.COMMAND_WORD -> new ScheduleCommandParser().parse(arguments);
         case AssignPiecesCommand.COMMAND_WORD -> new AssignPiecesCommandParser().parse(arguments);
+        case UnassignPiecesCommand.COMMAND_WORD -> new UnassignPiecesCommandParser().parse(arguments);
```

Comment:
Unrelated, but I think we should reorder the stuff in the parser to match our standard command order